docs/docs/new.md

```diff
@@ -0,0 +1,78 @@
+# Composable, declarative prompts
```

Comment:
Do we want to include this code in an actual example?

Reply:
Oops committed this by mistake, will remove